sbt_wrapper/Dockerfile

```diff
@@ -7,12 +7,15 @@ ENV SBT_VERSION 1.2.8
 ENV SBT_HOME /usr/local/sbt
 ENV PATH ${PATH}:${SBT_HOME}/bin
 
+RUN apt-get update
 RUN apt-get install -y \
```

Comment:
Minor: probably not a big deal for us and the way we build images, but in general I bundle these two together when writing a Dockerfile, i.e.

```
RUN apt-get update && \
    apt-get install -y \
...
```

Each declaration in a Dockerfile is a separate "layer", and Docker only rebuilds layers when the declaration changes. Having the `apt-get update` and the `apt-get install` on different lines means the two can drift, and you can end up in weird-to-debug situations where an `install` command fails because you're running a stale set of package updates.